applications/nrf5340_audio/src/audio/audio_sync_timer.h

```diff
@@ -61,4 +62,18 @@ uint32_t audio_sync_timer_i2s_frame_start_ts_get(void);
  */
 uint32_t audio_sync_timer_curr_time_get(void);
 
+/**
+ * @brief Blinks the LED used to show synchronization
```

Comment:
I suggest changing to: "Set timer compare value for when to clear or set the sync LED"